### DISCUSSION TITLE
Publishing native builds for Debian based systems

### DISCUSSION THREAD

**Discussion by CrispyToken** [PavelZinchenko/event-horizon-main#297] [Ideas]
In addition to the existing natively supported platforms (Windows and Android), it might be beneficial to consider publishing compiled builds for systems running Linux, specifically those based on Debian, since it is the most widely used distribution base today. 

This proposition mostly stems from personal motivation, since while currently builds targeting Windows can be launched and run using compatibility layers like Wine, this often leaves much to be desired and is generally not a stable and smooth experience. I have also encountered others in the community attempting to compile the project for themselves for Linux, however these builds obviously lack full functionality, namely the ability to use game saves created by the officially published builds or load compiled mod files.

Hence the idea is to allow for officially published builds allowing full functionality that targets this platform whether in the form of an installable .deb package or something else.

**Comment by juh9870** [comment]
I don't think .deb installer is needed, and may not even be compatible with other systems. Just a Linux binary should be enough.

**Reply by CrispyToken** [reply]
You are right, that might actually be simpler to do and also allow it to be compatible with any linux based system.

**Comment by Okunek9765** [comment]
Как я могу узнать, где можно найти коды на Event Horizon?

**Comment by PavelZinchenko** [comment]
I'm not familiar with Linux. I can just publish the same build I usually make for Steam.
